src/components/Storage/DesktopStorage/DesktopItemsList/Item/ListItem.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Flex,
+  Image,
+  Td,
+  Text,
+  Tr,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper,
+  Button,
+  ButtonGroup,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { Item } from '../../../../../mongo/models/item'
+import ProductButton from '../../../../UI/Custom Buttons/ProductButton/ProductButton'
+
+interface Props {
+  item: Item
+}
+
+const ListItem = ({ item }: Props) => {
+  const [quantity, setQuantity] = useState(0)
+  const [isEdit, setIsEdit] = useState(false)
+  return (
+    <Tr fontSize="14px" h="40px">
```

Comment:
To bym podzielił na dwa komponenty, bo jak ktoś to będzie musiał za miesiąc jakoś zmienić, to może zejść na zawał. A przynajmniej się skrzywić